src/observers/link_prefetch_observer.js

```diff
@@ -78,6 +78,8 @@ export class LinkPrefetchObserver {
           target
         )
 
+        fetchRequest.fetchOptions.priority = 'low'
```

Comment:
Minor nit: prefer double quotes for String literals.

```suggestion
        fetchRequest.fetchOptions.priority = "low"
```